source/_docs/automation/trigger.markdown

```diff
@@ -182,13 +182,13 @@ automation:
   trigger:
     - platform: numeric_state
       entity_id: sensor.temperature
-      # Optional
-      value_template: "{{ state.attributes.battery }}"
+      # If given, will trigger when the value of the given attribute for the given entity changes..
+      attribute: attribute_name
+      # ..or alternatively, will trigger when the value given by this evaluated template changes.
+      value_template: "{{ state.attributes.value - 5 }}"
```

Comment:
Actually, this one should be removed.

> the use of templates should be avoided if a pure YAML version is available.

<https://developers.home-assistant.io/docs/documenting/yaml-style-guide#templates>

Reply:
Hm, but don't they serve different purposes? The `value_template` property can do much more than the `attribute` property. If only a pure attribute is wanted, one should of course use `attribute`, but `attribute` doesn't work for the more advanced case I added below this one (calculating a difference between two attributes of the same entity).

Though I can agree that using a trivial example like this one (`- 5`) doesn't make sense, because `attribute` plus adjusting `above` / `below` works too.

Or am I missing something here?

Reply:
yes, they do, but this part of the documentation is about triggers, not about templates. We should avoid templating when possible. In this case, it isn't needed to talk about a template to explain what a trigger is.

Reply:
Ok, makes sense to some degree. So where should `value_template` be documented then? 🤔

Reply:
Oh darn, you are right... I'm sorry this is the page listing all triggers types 🤦